app/shared/services/contentful-content.service.ts

```diff
@@ -66,11 +77,11 @@ export class ContenfulContent {
    * @param slug
    * @returns {any}
    */
-  getNodePage(slug: string): Observable<NodePageContent> {
+  getNodePage(slug: string): Observable < NodePageContent > {
```

Comment:
ok, sounds good to me, this one is from pycharm auto formatting - you have right ... 
ok, now I know, when I was refactoring the code, I'd removed one bracket and then reformatted the code with pycharm, this is the reason why it looks strange. So it's my mistake, not my intent to have different formatting style.